config/beam_packet_merge.yaml

```diff
@@ -0,0 +1,69 @@
+##########################################
+#
+# beam_packet_merge.yaml
+#
+# Configuration for merging tracking extra 
+# beam receivers' raw packet to a more 
+# packed buffer format and write them to 
+# files. 
+#
+# Author: Jing Luo
+#
+##########################################
+---
+type: config
+log_level: debug2
```

Comment:
FYI: You can set the log level for any stage individually.  So if you don't want to see debug messages from `bufferRecv` you can set this to `info` or `warn` and add a `log_level: debug2` line into the block for the stage you want to see the extra debugging from.